Jenkinsfile

```diff
@@ -17,7 +17,7 @@ node('docker') {
 
     if (env.BRANCH_NAME == 'master') {
         stage('make') {
-            sh "make build; cd dist; ./build-for-openshift.sh"
+            sh "cd dist; ./build-for-openshift.sh"
```

Comment:
You can move `build-for-openshift.sh` from `dist`, so you don't have to do the change of directory in script again.

anyway, you can move most of the functionality from `build-for-openshift.sh` to spec file. Creating  `build-for-openshift.sh` was one of the hacks which were required to get running on OS V2.

I would keep in this script just creating an archive, downloading data-normalizer and creating srpm and rename script. However, I will be glad to hear your thoughs about this one.